CONTRIBUTING.md

```diff
@@ -171,6 +171,7 @@ Once the artifacts are built, you can start using them in your application by fo
 * `../../bin/flutter run --local-engine-src-path /path/to/engine/src --local-engine=ios_debug_unopt` or `--local-engine=ios_debug_sim_unopt` for simulator
   * If you are debugging crashes in the engine, you can connect the `LLDB` debugger from `Xcode` by opening `ios/Runner.xcworkspace` and starting the application by clicking the Run button (CMD + R).
   * To debug non crashing code, open Xcode with `ios/Runner.xcworkspace`, expand Flutter->Runner->Supporting Files->main.m in the Runner project. Put a breakpoint in main() then set your desired breakpoint in the engine in [lldb](https://lldb.llvm.org/tutorial.html) via `breakpoint set -...`.
+  * Note that this will set a `$LOCAL_ENGINE` variable in your `Generated.xcconfig` file for the Flutter application.  This variable will be set until you run `flutter run` again with either a different `--local-engine` option, or with none at all (which will unset it).  This is important to note if you plan to re-run your Flutter `Runner.xcodeproj` directly from Xcode.  In that case, you should also consider updating the custom run script that calls `$FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh` to also call that script with the `embed` parameter.
```

Comment:
Not clear in the last sentence what one is supposed to update xcode_backend with

Reply:
Hope that's clearer